airbyte/cloud/connections.py

```diff
@@ -392,6 +392,103 @@ def set_selected_streams(self, stream_names: list[str]) -> CloudConnection:
         self._connection_info = updated_response
         return self
 
+    # Enable/Disable
+
+    def enable(self) -> CloudConnection:
```

Comment:
Let's combine these into a single method to set enabled status. Let's also add an input arg to ignore no op, meaning don't raise if the status is already what we are asking for. Default behavior is to Not raise If the request turns out to be a no-op. 

Then, on top of that, add an implementation that's a simpler enabled property, calling the method with the default option to not raise if it's tried to set to something that's already existing. When retrieving this value, we will not use any cached values, because we won't know if another process or human has toggled the setting in between us checking it